### PR TITLE
Misc

### PR DESCRIPTION
**Pull request by jsoref**

### CODE REVIEW

File: config/default/kustomization.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ namespace: garo-system
 
 # Value of this field is prepended to the
 # names of all resources, e.g. a deployment named
-# "wordpress" becomes "alices-wordpress".
+# "wordpress" becomes "garo-wordpress".
```
Comment (davidkarlsen) [inline config/default/kustomization.yaml:6]: The change isn't really required, nor is it consistent. It's a generated explanation of how it works.

File: README.md [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
 # github-actions-runner-operator
 
-K8S operator for scheduling github actions runner pods.
+K8s operator for scheduling github actions runner pods.
```
Comment (davidkarlsen) [inline README.md:10]:
```suggestion
k8s operator for scheduling github actions runner pods.
```

Comment (jsoref) [reply README.md:10]: "Kubernetes (K8s) is an open-source system for automating deployment, scaling, and management of containerized applications." 
https://kubernetes.io/

The first letter gets capitalized when it's at the beginning of a sentence.

**Comment by davidkarlsen** [comment]
Fixed in 6d10e290ec2d6c9a04b2fc75525bc7a9cde62a91

**Pull request closed without merging**